internal/virtualenvironment/virtualenvironment_test.go

```diff
@@ -38,19 +37,6 @@ func teardown() {
 	rtMock.Close()
 }
 
-func TestActivateFailureAlreadyActive(t *testing.T) {
-	setup(t)
-	defer teardown()
-
-	os.Setenv(constants.ActivatedStateEnvVarName, "test")
-
-	venv := New(nil)
-	failure := venv.Activate()
-	require.NotNil(t, failure, "expected a failure")
-	assert.Equal(t, FailAlreadyActive, failure.Type)
-	assert.Equal(t, locale.Tr("err_already_active"), failure.Error())
-}
```

Comment:
Seeing as this broke and we weren't aware; can we ensure that we're still testing this? Just in the right place. Probably an integration test.